classes/object_manipulator/manipulator.php

```diff
@@ -84,58 +84,60 @@ public function __construct($filesystem, $config) {
     /**
      * Pushes files from local file system to remote.
      *
-     * @param  array $candidatehashes content hashes to push
      */
-    public function execute($objectrecords) {
-
+    public final function execute() {
         if (!$this->manipulator_can_execute()) {
             mtrace('Objectfs manipulator exiting early');
             return;
         }
 
-        if (count($objectrecords) == 0) {
-            mtrace('No candidate objects found.');
-            return;
-        }
+        $limit = 1000; // TODO: config option?
```

Comment:
Is there a point to this? I'd just remove this and let the default arg happen

If we add a config then I'd add into get_candidate_objects so that code can override it by setting the arg

Reply:
I wanted to make it more obvious